be/src/service/internal_service.cpp

```diff
@@ -50,7 +50,11 @@ using PromiseStatus = std::promise<Status>;
 using PromiseStatusSharedPtr = std::shared_ptr<PromiseStatus>;
 
 template <typename T>
-PInternalServiceImplBase<T>::PInternalServiceImplBase(ExecEnv* exec_env) : _exec_env(exec_env) {}
+PInternalServiceImplBase<T>::PInternalServiceImplBase(ExecEnv* exec_env)
+        : _exec_env(exec_env),
+          // Now, only get_info is processed by _async_thread_pool, and only needs a small number of threads.
+          _async_thread_pool("async_thread_pool", config::internal_service_async_thread_num,
```

Comment:
Is the queue length a bit small?

Reply:
The default value of `internal_service_async_thread_num` is changed to 10.